public/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -259,28 +276,26 @@ Light client mode downloads all block headers, block data, and verifies some ran
 
 **NB** Light sync does not yet work with proof-of-stake Ethereum - new versions of light sync should ship soon!
 
-[More on light clients](/developers/docs/nodes-and-clients/light-clients/)
+[More on light clients](/developers/docs/nodes-and-clients/light-clients/).
```

Comment:
This is a convention used commonly on the site, without a trailing hard stop; please revert these

```suggestion
[More on light clients](/developers/docs/nodes-and-clients/light-clients/)
```